doc/manual/source/command-ref/nix-instantiate.md

```diff
@@ -112,6 +112,11 @@ standard input.
   When used with `--eval`, print the resulting value as an JSON
   representation of the abstract syntax tree rather than as a Nix expression.
 
+- `--replace-eval-errors`
+
+  When used with `--eval` and `--json`, replace any evaluation errors with the string
+  `"«evaluation error»"`.
```

Comment:
This isn't really what it does. It should say something like: any attribute that throws an evaluation error is represented by a JSON property with the string value `«evaluation error»`.